homeassistant/components/triggercmd/config_flow.py

```diff
@@ -0,0 +1,66 @@
+"""Config flow for TRIGGERcmd integration."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+import jwt
+from triggercmd import client
+import voluptuous as vol
+
+from homeassistant import config_entries, exceptions
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN  # pylint:disable=unused-import
```

Comment:
```suggestion
from .const import DOMAIN
```

Should not be required